src/CitizenProject/Voter/CreateCitizenProjectVoter.php

```diff
@@ -17,30 +17,27 @@ public function __construct(CitizenProjectManager $manager)
         $this->manager = $manager;
     }
 
-    public function vote(TokenInterface $token, $subject, array $attributes)
+    protected function supports($attribute, $subject): bool
```

Comment:
the method signature needs to be the same as its parent:
protected function supports($attribute, $subject)